module/VuFind/src/VuFind/ILS/Driver/MultiBackend.php

```diff
@@ -1231,7 +1171,7 @@ protected function getSourceFromParams(
      *
      * @param string $source The source name of the driver to get.
      *
-     * @return mixed  On success a driver object, otherwise null.
+     * @return mixed On success a driver object, otherwise null.
```

Comment:
Out of scope for this PR, but maybe something @EreMaijala should consider in a follow-up -- I see that $source can be false-y based on the code below, but the comment and type here don't seem to reflect that.